modules/bim/app/models/bim/ifc_models/ifc_model.rb

```diff
@@ -27,7 +27,7 @@ class IfcModel < ActiveRecord::Base
           delete_attachment name
           call = ::Attachments::CreateService
             .bypass_whitelist(user: User.current)
-            .call(file: file, container: self, filename: file.original_filename, description: name)
+            .call(file: file, container: self, filename: File.basename(file.path), description: name)
```

Comment:
For an uploaded file, this is not the same. The original filename will be provided by the filename of the user (which is what you probably want), whereas the filepath is a temporary filename. For example:

```
[9] pry(#<Bim::IfcModels::SetAttributesService>)> ifc_attachment.original_filename
=> "202107010001_berlin_schoeneweide_2001.ifc"
[10] pry(#<Bim::IfcModels::SetAttributesService>)> File.basename ifc_attachment.path
=> "RackMultipart20210924-65661-1k948ph.ifc"
```

We could check if the given file responds to original_filename and use that, and fallback to the file name instead. Or you wrap the XKT attachment in a ` Rack::Multipart::UploadedFile.new('path')`